web/src/main/kotlin/no/nav/su/se/bakover/web/routes/sak/SakRoutes.kt

```diff
@@ -275,29 +282,77 @@ internal fun Route.sakRoutes(
     post("$SAK_PATH/{sakId}/fritekstDokument/lagreOgSend") {
         authorize(Brukerrolle.Saksbehandler) {
             call.withSakId { sakId ->
-                call.withBody<DokumentBody> { body ->
-                    val res = sakService.lagreOgSendFritekstDokument(
-                        OpprettDokumentRequest(
-                            sakId = sakId,
-                            saksbehandler = call.suUserContext.saksbehandler,
-                            tittel = body.tittel,
-                            fritekst = body.fritekst,
-                            distribueringsadresse = body.adresse?.toDomain(),
-                            distribusjonstype = body.distribusjonstype.toDomain(),
-                        ),
-                    )
+                when (call.request.headers["content-type"]?.contains("multipart/form-data")) {
+                    /**
+                     * Dersom requesten er multipart, har dem lagt på en allerede generert PDF, og vi skal bare
+                     * journalføre, og distribuere denne.
+                     */
+                    true -> {
+                        val parts = call.receiveMultipart().readAllParts()
```

Comment:
Kunne trukket ut i egen funksjon. Vanskeligere å lese en sånn lang when med mange linjer i for hver case.